files/cf-exclude-include.json

```diff
@@ -15,7 +15,8 @@
     "magnesium-extras",
     "skin-layers-3d",
     "textrues-rubidium-options",
-    "torohealth-damage-indicators"
+    "torohealth-damage-indicators",
+    "dynamiclights-reforged"

```

Comment:
I'd like to keep this list alphabetical since it's going to get longer, so please shift this one up to line 5.